### PR TITLE
HPCC-21593 Add Disk Usage Dashboard

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq / @kunalaswani please review (Note:  Its a draft PR as it is dependent on the next release of the Viz Framework)

**Comment by GordonSmith** [comment]
@AttilaVamos - I suspect your smoke test should ignore PRs which are marked as "Draft"?

**Comment by AttilaVamos** [comment]
@GordonSmith I implemented your request. Please push a new commit to check it. Thanks.

**Comment by richardkchapman** [comment]
@AttilaVamos Not sure I necessarily agree - one of the possible uses for draft PRs is to get the smoketests to run

**Comment by jakesmith** [comment]
Perhaps we could use a special tag in the title instead. e.g. "PREVIEW:" or similar - to indicate it's a PR for review/sharing only, but not ready to smoketest.

**Comment by GordonSmith** [comment]
Or maybe one of the checkboxes in the default template could be used "run smoketest" defaulted on?

**Comment by GordonSmith** [comment]
@miguelvazq / @kunalaswani this is ready for review + pulling now...

**Comment by kunalaswani** [comment]
code looks good

File: esp/src/lws.config.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
-const debugServerIP = "192.168.99.103";
```
Comment (richardkchapman) [inline esp/src/lws.config.js]: Not sure these should be in a checked in file?

Comment (AlexSmith) [reply esp/src/lws.config.js]: I concur (I am not sure either) - but it is really handy having them...

Comment (AlexSmith) [reply esp/src/lws.config.js]: (I can compromise by only keeping the 192.* ones?)

**Pull request merged**